### PR TITLE
Remove most references to old gpucomputing site and point at new home for CUDA training material

### PR DESCRIPTION
**Pull request by willfurnass**
Also, RSE team not necessarily want to be first point of contact for GPU-related support queries and for maintaining centrally-provided software stacks (RSE folks inc @robadob: let me know if you disagree!)

### CODE REVIEW

File: sharc/DeepLearningShARC.rst [hunk 1]
```diff
@@ -38,4 +38,4 @@ The Nvidia DGX-1 is the world's first Deep Learning supercomputer. It is availab
 Training Materials
 ------------------
 
-* `Introducting to Deep Learning using Caffe on ShARC's DGX-1 by GPUComputing@Sheffield <http://gpucomputing.shef.ac.uk/education/cuda/>`_
+* `Introducting to Deep Learning using Caffe on ShARC's DGX-1 by GPUComputing@Sheffield <https://rse.shef.ac.uk/training/com4521//>`_
```
Comment (ptheywood) [inline sharc/DeepLearningShARC.rst:41]: This link title doesn't match the destination.

File: sharc/software/libs/cuda.rst [hunk 2]
```diff
@@ -155,8 +155,8 @@ but file locking was not previously enabled on the (`Lustre <http://lustre.org/>
 CUDA Training
 -------------
 
-`GPUComputing@sheffield <http://gpucomputing.shef.ac.uk>`_ provides 
-a self-paced `introduction to CUDA <http://gpucomputing.shef.ac.uk/education/cuda/>`_ training course.
+The Research Software Engineering team provide
+a self-paced `introduction to CUDA <https://rse.shef.ac.uk/training/com4521>`_ training course.
```
Comment (ptheywood) [inline sharc/software/libs/cuda.rst:159]: Not sure if this is appropriate to call the self-paced intro to CUDA given it's linking to the 4th year COM Module, though not sure we have anything else that is appropriate as a recent equivalent?

Comment (Robadob) [reply sharc/software/libs/cuda.rst:159]: The lecture notes should be available to anyone with a university google account, there's also Paul's lecture recordings (on Kaltura) linked from the page.

No harm people contacting me from that page if they have more niche requirements.

File: bessemer/software/libs/cuda.rst [hunk 3]
```diff
@@ -252,8 +252,8 @@ but file locking was not previously enabled on the (`Lustre <http://lustre.org/>
 CUDA Training
 -------------
 
-`GPUComputing@sheffield <http://gpucomputing.shef.ac.uk>`_ provides
-a self-paced `introduction to CUDA <http://gpucomputing.shef.ac.uk/education/cuda/>`_ training course.
+The Research Software Engineering team provide
+a self-paced `introduction to CUDA <https://rse.shef.ac.uk/training/com4521>`_ training course.
```
Comment (Robadob) [inline bessemer/software/libs/cuda.rst:256]: Same feedback as the same text Pete commented on

File: sharc/GPUComputingShARC.rst [hunk 4]
```diff
@@ -123,5 +123,4 @@ GPU-enabled Software
 Training materials
 ^^^^^^^^^^^^^^^^^^
 
-* `Introduction to CUDA by GPUComputing@Sheffield <http://gpucomputing.shef.ac.uk/education/cuda/>`_
-* `Introducting to Deep Learning using Caffe on ShARC's DGX-1 by GPUComputing@Sheffield <http://gpucomputing.shef.ac.uk/education/cuda/>`_
+* `Introduction to CUDA by the Research Software Engineering (RSE) team <https://rse.shef.ac.uk/training/com4521>`_
```
Comment (Robadob) [inline sharc/GPUComputingShARC.rst:126]: I can tweak the web-page to add a note that content *should* be suitable to follow self-paced, get in contact etc.

File: sharc/software/libs/cuda.rst (hunk 2 above)
Comment (willfurnass) [reply sharc/software/libs/cuda.rst]: Reworded to state is a teaching module.

File: bessemer/software/libs/cuda.rst (hunk 3 above)
Comment (willfurnass) [reply bessemer/software/libs/cuda.rst]: Reworded to state is a teaching module.

File: sharc/GPUComputingShARC.rst (hunk 4 above)
Comment (willfurnass) [reply sharc/GPUComputingShARC.rst:126]: Would be nice if you're happy doing so

**Comment by Robadob** [comment]
> RSE folks inc @Robadob: let me know if you disagree!

I'm personally happy to waste time answering questions that fit my skillset (it's people above me that always say RSE team shouldn't touch topdesk), if I wasn't the COM4521 mailing list wouldn't have been so useful over the years.

I expect 99% of "GPU" questions are actually infrastructure (where to access GPU) or HPC (how to use x software). Presumably any CUDA or abstract GPU/theory questions would already be directed our way (via code clinics?).

File: bessemer/GPUComputingBessemer.rst [hunk 5]
```diff
@@ -150,5 +150,5 @@ Specifications per A100 node:
 Training materials
 ------------------
 
-* `Introduction to CUDA by GPUComputing@Sheffield <https://gpucomputing.shef.ac.uk/education/cuda/>`_
+* `Introduction to CUDA by the Research Software Engineering (RSE) team <https://rse.shef.ac.uk/training/com4521>`_
```
Comment (Robadob) [inline bessemer/GPUComputingBessemer.rst:153]: Text hasn't been updated here?

File: stanage/GPUComputingStanage.rst [hunk 6]
```diff
@@ -102,4 +102,4 @@ GPU-enabled Software
 Training materials
 ------------------
 
-`Introduction to CUDA by GPUComputing@Sheffield <https://gpucomputing.shef.ac.uk/education/cuda/>`_
+* `Introduction to CUDA by the Research Software Engineering (RSE) team <https://rse.shef.ac.uk/training/com4521>`_
```
Comment (Robadob) [inline stanage/GPUComputingStanage.rst:105]: Or here (not sure if you want to).

**Comment by twinkarma** [comment]
@willfurnass I'd also probably just remove any references to Caffe and Torch (the Lua version) as they are effectively dead. Last Caffe release was 2017 and Torch has not had an update since 2019.

**Comment by willfurnass** [comment]
@twinkarma Ta.  I've added a note to discourage people from using Caffe.  There's already a similar note re Lua Torch.

File: stanage/GPUComputingStanage.rst (hunk 6 above)
Comment (willfurnass) [reply stanage/GPUComputingStanage.rst]: Done

**Pull request merged**